docs/userGuide/syntax/links.mbdf

```diff
@@ -45,15 +45,44 @@ MarkBind home is at [here][1]
 
 <div id="intraSiteLinks">
 
-Links to files of the generated site (e.g., an HTML page or an image file) should be specified as absolute paths and should start with {{ showBaseUrlCode }} (which represents the root directory of the project).
+Links to files of the generated site (e.g., an HTML page or an image file) can be specified either as relative paths or absolute paths.
 
+Absolute paths:
 <div class="indented">
+Links should start with {{ showBaseUrlCode }} (which represents the root directory of the project). 
 
 {{ icon_example }} Here's how to specify a link to (1) a page, and (2) an image, using the {{ showBaseUrlCode }}:
 
-1. <code>Click [here]({<span></span>{ baseUrl }}/userGuide/reusingContents.html).</code>
+1. <code>Click [here]({{ showBaseUrlCode }}/userGuide/reusingContents.html).</code>
 2. `![](`{{ showBaseUrlCode }}`/images/preview.png)`
 
+</div>
+
+Relative paths:
+
+<div class="indented">
+Links to files can also be specified relative to the file that includes it.
+
+{{ icon_example }} Assuming that we have the following folder structure:
+```
+/
+- index.md
+- sub/
+- - img.png
+- - subsite.md
+```
```

Comment:
Make it look like this?
![image](https://user-images.githubusercontent.com/1673303/53690114-4815e680-3d9f-11e9-81f2-955612503dde.png)

Reply:
That's a nice way to represent the file structure. Will do that. Thanks!